src/app/course/_test/course-question-snippet/course-question-snippet.component.spec.ts

```diff
@@ -64,6 +79,87 @@ describe('CourseQuestionSnippetComponent VALID EVENT', () => {
         expect(component.highlight('Partially Solved')).toEqual('highlight-warning');
         expect(component.highlight('Wrong')).toEqual('highlight-danger');
     });
+
+    it ('report should be deleted', () =>{
+        component.deleteReport();
+        expect(component.reportUQJ.report).toEqual({} as QuestionReport);
```

Comment:
why not create a new instance of the report? is there a reason to cast this here?

Reply:
This was the only way to instantiate an empty QuestionReport object that we know of.